### ISSUE TITLE
Accessibility bug: cannot scroll through timeline

### ISSUE THREAD

**Issue by IanGClifton** [android/socialite#6]
Accessibility focus does not allow navigating through the timeline correctly. If an image is displaying (e.g., the sheep reply), swiping right does not move away from the Timeline text image. Swiping right rapidly twice can sometimes move the focus to the animal name and swiping right again will scroll to the next item, but focus is moved to the Timeline text image again.

**Comment by calren** [comment]
@SigmanZero I can take a stab at looking at this one if you haven't started already.

**Comment by calren** [comment]
After some exploration and experimentation:

1. The content of item in the timeline should be grouped together: `.semantics(mergeDescendants = true) {}`
2. Using layout inspector, It seems like navigation isn't working correctly because the top app bar is laid out **after** the timeline contents.

**Comment by calren** [comment]
Narrowed down the issue to https://github.com/android/socialite/blob/main/app/src/main/java/com/google/android/samples/socialite/ui/home/timeline/Timeline.kt#L109-L116. 

I think because re-composition happens on every scroll to update the video in the player, the timeline gets recomposed and the ordering between the timeline / the top bar gets switched.

**Comment by calren** [comment]
This only happens if there is more than 1 media item in the timeline. If there is only one picture in the timeline, TalkBack navigation works fine.

**Comment by calren** [comment]
possibly related? https://issuetracker.google.com/333670357
